src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -1,47 +1,354 @@
+            try {
+                actions = getInstructorFeedbackSessionActions(session, false, 
+                                                              instructors.get(courseId),
+                                                              getCourseIdSectionNamesMap(sessions).get(courseId));
+            } catch (EntityDoesNotExistException e) {
+                // nothing
+            }
+            
+            ElementTag elementAttributes ;
+            if (session.courseId.equals(courseIdForNewSession) && session.feedbackSessionName.equals(feedbackSessionNameForSessionList)) {
+                elementAttributes = new ElementTag("class", "sessionsRow warning");
+            } else {
+                elementAttributes = new ElementTag("class", "sessionsRow");
+            }
+            
+            rows.add(new FeedbackSessionsTableRow(courseId, name, tooltip, status, href, recent, actions, elementAttributes));
+        }
+        
+        return rows;
+    }
+    
+    public FeedbackSessionsTable getFsList() {
+        return fsList;
+    }
+    
+    public FeedbackSessionsForm getNewFsForm() {
+        return newFsForm;
+    }
+    
+    public FeedbackSessionsCopyFromModal getCopyFromModal() {
+        return copyFromModal;
+    }
+
+    private ArrayList<ElementTag> getTimeZoneOptionsAsElementTags(FeedbackSessionAttributes fs){
+        return getTimeZoneOptionsAsElementTags(fs == null ? 
+                                               Const.DOUBLE_UNINITIALIZED : 
+                                               fs.timeZone);
+    }
+
+    
+    private ArrayList<ElementTag> getGracePeriodOptionsAsElementTags(FeedbackSessionAttributes fs) {
+        return getGracePeriodOptionsAsElementTags(fs == null ? 
+                                                  Const.INT_UNINITIALIZED : 
+                                                  fs.gracePeriod);
+    }
+    
+    /**
+     * Creates a list of options (STANDARD and TEAMEVALUATION). If defaultSessionType is null,
+     *     TEAMEVALUATION is selected by default.
+     * @param defaultSessionType  either STANDARD or TEAMEVALUATION, the option that is selected on page load
+     */
+    private ArrayList<ElementTag> getFeedbackSessionTypeOptions(String defaultSessionType) {
+        ArrayList<ElementTag> result = new ArrayList<ElementTag>();
+        
+        ElementTag standardFeedbackSession = createOption("Session with your own questions", 
+                                                          "STANDARD", defaultSessionType != null 
+                                                          && defaultSessionType.equals("STANDARD"));
+        ElementTag evaluationFeedbackSession = createOption("Team peer evaluation session", 
+                                                            "TEAMEVALUATION", defaultSessionType == null 
+                                                            || defaultSessionType.equals("TEAMEVALUATION")); 
```

Comment:
Should help readability if the two parts of the boolean condition are on the same line, or if the 2nd String is placed on the 1st line (saving the 2nd and 3rd lines for the boolean condition alone).